recipes/devel/cmake.yaml

```diff
@@ -68,5 +78,14 @@ multiPackage:
         packageScript: |
             installPackageTgt "$1/install/"
 
+    win64:
+        buildVars: [PKG_VERSION]
+        buildScript: |
+            mkdir -p install/usr
+            rsync -aH --exclude 'doc/' --exclude 'man/' "$1/cmake-${PKG_VERSION}-win64-x64/" install/usr/
+
+        packageScript: |
+            cp -a $1/install/* .
```

Comment:
I think you should put this all into a separate `recipes/devel/cmake-win64.yaml` recipe. The windows CMake recipe has nothing in common with the regular one except the name :laughing:

Reply:
I just liked, that this was possible in this way. :D 
But of course, creating a new recipe would have some advantages.
e.g. we can leave the BOB_HOST_PLATFORM stuff away.
I'll fix this.